control-plane/pkg/security/secret.go

```diff
@@ -173,15 +173,15 @@ func sslConfig(protocol string, data map[string][]byte) kafka.ConfigOption {
 			}
 
 			if !skipClientAuth {
-				userKeyCert, ok := data[UserKey]
+				userKeyCert, ok := firstNonEmptyValueOf(data, UserKey, "tls.key")
```

Comment:
Can we use constants instead of hardcoded strings?

Reply:
Okay!

Reply:
Done!